core/controllers/admin.py

```diff
@@ -1889,6 +1925,9 @@ class UpdateBlogPostHandler(
 
     @acl_decorators.can_access_admin_page
     def put(self) -> None:
+        """Handles PUT requests to update the author and published date of
+        a blog post.
```

Comment:
Maybe, "Updates the author and published date of a blog post". you don't have to add "Handles PUT requests"

Reply:
Alright.